Makefile

```diff
@@ -88,7 +88,7 @@ BASE_IMAGE_UUID = 502eeef2-8267-489f-b19c-a206906f57ef
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SDC UFDS
 BUILDIMAGE_DO_PKGSRC_UPGRADE = true
-BUILDIMAGE_PKGSRC = haproxy postgresql96-client-9.6.24nb1
+BUILDIMAGE_PKGSRC = coreutils haproxy postgresql96-client-9.6.24nb1
```

Comment:
I don't see what obviously requires adding coreutils.  Clearly I'm missing something...

Reply:
Needed for the `base64` command that's used in `ufds-reconcile-data`. It's actually already installed, but this is just making it explicit.